Refs #33604 -- allow the cache `key_prefix` to be a callable (#15553)
django/django: docs/releases/4.1.txt

Cache
~~~~~

> * The ``key_prefix`` argument of the
>   :func:`~django.views.decorators.cache.cache_page` decorator can now be a
>   callable.

Reviewer: ```suggestion
* :func:`~django.views.decorators.cache.cache_page` decorator now accepts a
  callable for ``key_prefix``.
```
